Security_Engine/Query/ViewCone.cs

```diff
@@ -48,7 +48,7 @@ public static PolyCurve ViewCone(this CameraDevice cameraDevice)
             Point targetLocation = cameraDevice.TargetPosition;
             double radius = targetLocation.Distance(cameraLocation);
             double horizontal = cameraDevice.HorizontalFieldOfView;
```

Comment:
```suggestion
            double fieldOfViewArcLength = cameraDevice.HorizontalFieldOfView;
```

Reply:
To be more precise, I used `coneArcLength`